core/src/components/picker-column/picker-column.tsx

```diff
@@ -185,6 +204,8 @@ export class PickerColumnCmp implements ComponentInterface {
         button.classList.remove(PICKER_OPT_SELECTED);
       }
     }
+
+    this.didColChange = false;
```

Comment:
Given that `didColChange` is only needed for `componentDidUpdate`, would it make more sense to set this to `false` in `componentDidUpdate`?